### PR TITLE
php folder structure updated

### PR DESCRIPTION
**Pull request by saifaliaxioned**

### CODE REVIEW

File: docs/learning/php/code-review-checklist.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+16.3 | There MUST be one blank line after the all use block. | For example:<br />```<?php namespace Vendor\Package;use FooClass;use BarClass as Bar;use OtherVendor\OtherPackage\BazClass;```
+17 | **Extends and Implements**
+17.1 | Declaration | The extends and implements keywords MUST be declared on the same line as the class name.
+17.2 | Class Declaration | The opening brace for the class MUST go on its own line; the closing brace for the class MUST go on the next line after the body.
+18 | **Properties**
+18.1 | Visibility MUST be declared on all properties. | Are the properties defined by prefixing the declaration with the keywords public, protected or private?
+18.2 | Avoid var keyword for property | Has var keyword used to declare a property?
+18.3 | Avoid more than one property per statement | Are there more than one property declared per statement?
+18.4 | Property name | Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.
+18.5 | A property declaration looks like the following. | Example:<br />````<?php namespace Vendor\Package; class ClassName { public $foo = null; }```
```
Comment (pkaranaxioned) [inline docs/learning/php/code-review-checklist.md:51]: Remove extra ` training

File: docs/learning/php/code-review-checklist.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+sidebar_label: Code Review Checklist
+sidebar_position: 2
+tags: [PHP code review, PHP checklist]
+---
+
+# Checklist
+
+Sr No. | Check | Description |
+:-: | --- | ---  
+1 | Backend Validation | Does the validation present in .php file?
```
Comment (pkaranaxioned) [inline docs/learning/php/code-review-checklist.md:13]: Make heading bold

File: docs/learning/php/code-review-checklist.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+id: php-checklist
+title: PHP Code Review Checklist
+sidebar_label: Code Review Checklist
+sidebar_position: 2
+tags: [PHP code review, PHP checklist]
```
Comment (pkaranaxioned) [inline docs/learning/php/code-review-checklist.md:6]: Add More tags

File: docs/learning/php/php.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
-position: 4
-id: php
+id: php-training
 title: PHP Training Template
-sidebar_label: PHP
+sidebar_label: Training Template
+sidebar_position: 1
 tags: [PHP, PHP training]
```
Comment (pkaranaxioned) [inline docs/learning/php/php.md:6]: Add more tags

**Review by smehboobprdxn — APPROVED** [review]

**Pull request merged**